Formula/portable-ruby.rb

```diff
@@ -14,19 +14,22 @@ class PortableRuby < PortableFormula
 
   depends_on "makedepend" => :build
   depends_on "pkg-config" => :build
+  depends_on "portable-zlib" => :build unless OS.mac?
   depends_on "portable-readline" => :build
   depends_on "portable-libyaml" => :build
   depends_on "portable-openssl" => :build
   depends_on "portable-ncurses" => :build if OS.linux?
 
   def install
-    ENV.append "LDFLAGS", "-Wl,-search_paths_first"
-
+    zlib = Formula["portable-zlib"]
     readline = Formula["portable-readline"]
     libyaml = Formula["portable-libyaml"]
     openssl = Formula["portable-openssl"]
     ncurses = Formula["portable-ncurses"]
 
+    ENV.append "LDFLAGS", "-Wl,-search_paths_first"
+    ENV.append "LDFLAGS", "-lz -L#{zlib.opt_prefix/"lib"}" unless OS.mac?
```

Comment:
I'm not sure. But it is seem that you can add zlib lib using `args << "--with-opt-dir=#{paths.join(":")}"` in below, instead of hacking the LDFLAGS. Can you test it? @alyssais